Account/FxAClient10.swift

```diff
@@ -212,6 +255,40 @@ public class FxAClient10 {
         return nil
     }
 
+    private class func statusResponseFromJSON(json: JSON) -> FxAStatusResponse? {
```

Comment:
Question: Do these need to be defined as class methods? Looks like we're always using them within this class and inside non-class methods.